style.css

```diff
@@ -417,5 +451,149 @@ hr {
   width: 140px;
   float: left;
   margin-top: 1px;
-  margin-right: 220px;
+  margin-right: 200px;
+}
+
+@media (min-width: 748px) {
+  section {
+    background-position: 20px 0;
+  }
+
+  .header-section {
+    width: 100%;
+  }
+
+  .main-part {
+    min-width: 1100px;
+    padding-top: 160px;
+    padding-bottom: 260px;
+  }
+
+  .my-logo {
+    margin-left: 80px;
+  }
+
+  .navigation-links {
+    display: inline;
+    margin-right: 40px;
+  }
+
+  .navigation-right {
+    display: none;
+  }
+
+  .welcome-message {
+    margin-left: 160px;
+  }
+
+  .about-me-main {
+    font-size: 16px;
+    margin-top: -20px;
+    background-image: none;
+    margin-left: 160px;
+    max-width: 800px;
+  }
+
+  .about-me {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 40px;
+  }
+
+  .about-text {
+    margin-left: 100px;
+  }
+
+  .about-me-text {
+    margin-left: 100px;
+    margin-top: -10px;
+  }
+
+  .connect {
+    margin-left: 160px;
+    display: block;
+  }
+
+  .connect-bottom {
+    margin-left: 100px;
+    display: block;
+  }
+
+  .social-media-icons {
+    margin-left: 120px;
+    margin-top: 10px;
+  }
+
+  .social-media-button {
+    margin-left: 60px;
+  }
+
+  .portfolio {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .portfolio-border {
+    width: 80%;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .portfolio-image {
+    margin-top: 16px;
+    width: 90%;
+    height: 90%;
+    margin-left: 15px;
+  }
+
+  .project-section-reverse {
+    order: -1;
+  }
+
+  .btn--resume {
+    margin-left: 90px;
+  }
+
+  .elements {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+  }
+
+  .side-language {
+    display: flex;
+    justify-content: flex-start;
+    gap: 60px;
+  }
+
+  .contact-form {
+    background-image: url("/Images/contact-form-background-shapes.png");
+  }
```

Comment:
- [ ] [OPTIONAL] start your path for image with `./` instead of `/` because when you deploy your project the image will not appear.